xml/System.Buffers/ArrayPool`1.xml

```diff
@@ -25,7 +25,7 @@
   <Interfaces />
   <Docs>
     <typeparam name="T">The type of the objects that are in the resource pool.</typeparam>
-    <summary>Provides a resource pool that enables reusing instances of type <see cref="T:T[]" />.</summary>
+    <summary>Provides a resource pool that enables reusing instances of type T.</summary>
```

Comment:
I think `T[]` makes more sense in this context, rather than just T.

i.e. what the output looks like today: https://docs.microsoft.com/en-us/dotnet/api/system.buffers.arraypool-1?view=netcore-3.0
![image](https://user-images.githubusercontent.com/6527137/68621179-ef493280-0483-11ea-9525-82414722ea81.png)

Reply:
Ok, changed to T[].